hosted_control_planes/hosted-control-planes-release-notes.adoc

```diff
@@ -133,6 +133,15 @@ To resolve these issues, configure the DNS server settings for a dual stack netw
 ** You created a hosted cluster on the Agent platform through the {mce} console by using the default hosted cluster cluster namespace.
 ** You created a hosted cluster through the command-line interface or API by specifying the hosted cluster namespace to be the same as the hosted cluster name.
 
+//OCPBUGS-42779
+* If you configured a cluster-wide proxy for a new hosted cluster, the deployment of that cluster might fail because worker nodes cannot reach the Kubernetes API server when a cluster-wide proxy is configured. To resolve this problem, in the configuration file for the hosted cluster, add any of the following information to the `noProxy` field so that the traffic from the data plane to the Kubernetes API server skips the proxy:
+
+** The external API address.
+** The internal API address. The default is `172.20.0.1`.
+** The phrase `"kubernetes"`.
```

Comment:
In the `noProxy` field, if the user specifies the phrase `kubernetes`, do they need to include the double-quotation marks? For example, in the config file, does it look like this?
```
...
noProxy: kubernetes
...
```
or does it look like this?
```
...
noProxy: "kubernetes"
...
```

Reply:
Either way works. In general in YAML you don't need to quote strings unless its type is ambiguous

Reply:
Thank you!